entry/hvigorfile.js

```diff
@@ -0,0 +1,2 @@
+// Script for compiling build behavior. It is built in the build plug-in and cannot be modified currently.
+module.exports = require('@ohos/hvigor-ohos-plugin').legacyHapTasks
```

Comment:
Looks copyright need to be added in all js / ets files, please check with Leela and add